service/dap/server.go

```diff
@@ -1702,7 +1741,7 @@ func (s *Server) convertVariableWithOpts(v *proc.Variable, qualifiedNameOrExpr s
 		if v.Len > int64(len(v.Children)) { // Not fully loaded
 			if v.Base != 0 && len(v.Children) == 0 { // Fully missing
 				value = reloadVariable(v, qualifiedNameOrExpr)
-			} else { // Partially missing (TODO)
```

Comment:
I don't think that I actually addressed the reloading of the other variables in this function. I am going to leave it for now and can address this in a separate PR that looks at reloading a variable instead of calling EvalVariableInScope if thats alright.

Reply:
Sorry I was not more clear. I thought it could be a good idea to update the how-to details of the TODO given what you went through exposing some of the internal proc hooks. I agree that any reloading should be done in a separate PR.